README.md

```diff
@@ -93,6 +102,20 @@ Ansible Best Practices
   Avoid duplicating the remote full path in the role directory, however, as that creates unnecessary depth in
   the file tree for the role. Grouping sets of similar files into a subdirectory of `templates` is allowable,
   but avoid unnecessary depth to the hierarchy.
+* Example playbooks are to live in the root directory, not in a
```

Comment:
I think a bit more structure is needed. Those items added below were under the "Documentation conventions" header in the working doc.

Reply:
This appears to be my fault. This PR should only be the python bits. I copied / pasted too much from the orig google doc. Stand by

Reply:
Updated.